src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,80 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+
 public class WorkWithFile {
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
+    private static final String RESULT = "result";
+    private static final String DELIMITER = ",";
+    private static final int DEFAULT_VALUE = 0;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        Map<String, Integer> operationCounts = getInfoFromFile(fromFileName);
+        saveStatistics(toFileName, getStatistics(operationCounts));
```

Comment:
```suggestion
        List<String> dataFromFile = readFile(fromFileName);
        String report = createReport(dataFromFile);
        writeToFile(report, toFileName);
```
Do not mix file reading with business logic in one method